src/Core/PostgreSQLProtocol.h

```diff
@@ -724,8 +724,9 @@ class DataRow : BackendMessage
     Int32 size() const override
     {
         Int32 sz = 4 + 2; // size of message + number of fields
```

Comment:
Is this size in bytes? Why not to use `sizeof` some structure here? What will be if we change protocol?

Reply:
Yes, in bytes. But it is not a change in protocol, this is how it should have been by protocol.